### ISSUE TITLE
item spawn exploit

### ISSUE THREAD

**Issue by KoromaruKoruko** [bsalha1/Printer#24]
People are managing to use printer to give themselves items,
version: 3.0 
mc version: 1.8.8 on "Elapsed spigot" a paper fork.

in short.
go into print mode.
put items into inventory.
exit print mode, and cause network lag while doing so.
spam drop items from inventory.

my guess is that due to the network delay the drop item event is being fired after printer thinks the player is out of print mode.
so in short. server accounts for lag, printer does not. items given by printer are now real items on the ground.

**Comment by KoromaruKoruko** [comment]
one way I thought of being able to properly go around this would be to add nbt to items pulled from the creative inventory while in printer mode. and then check to see if that item has that nbt on a drop item event. then if it does, just deleting that item.

**Comment by KoromaruKoruko** [comment]
fixed
